CMakeLists.txt

```diff
@@ -33,6 +34,14 @@ if(FORCE_COLORED_OUTPUT)
   endif()
 endif()
 
+include(GNUInstallDirs)
+if(FORCE_PORTABLE_INSTALL)
+  set(CMAKE_INSTALL_BINDIR ".")
+  set(CMAKE_INSTALL_LIBDIR ".")
```

Comment:
If I understand correctly we use these variables to prepare for a future possible system installation of the game? This looks like an anti-pattern to me to include them just to override them after.

Reply:
We should never "install" the game in the traditional sense imho.

Reply:
if we ever want to package for Linux distributions it can, but I don't see this being implemented in the near future.

Reply:
@Lgt2x is correct about my intentions about future installation game on Linux systems. These variables redefined only for "portable variant", which finds many applications such correct way to build and install all required targets to assemble portable game installation instead using custom commands in CMakeLists.txt.